### ISSUE TITLE
ID conflict

### ISSUE THREAD

**Issue by timosachsenberg** [HUPO-PSI/psi-ms-CV#59]
After an update to the new obo we get a lot of:
Warning: while loading term 'MS:1003130' of CV 'MS': parent term name 'proteoform-level Q-value' and id 'MS:1003127' differ.
warnings when reading mzMLs.
Looking at the terms how they stand right now I think it could be an issue that MS:1003130 is a MS:1003127 ! proteoform-level Q-value but MS:1003127 has changed to ProSight:spectral E-value
see:
```
[Term]
id: MS:1003127
name: ProSight:spectral E-value
def: "ProSight spectrum-level E-value." [PSI:MS]
xref: value-type:xsd\:double "The allowed value-type for this CV term."
is_a: MS:1002353 ! PSM-level e-value
is_a: MS:1001143 ! PSM-level search engine specific statistic
relationship: has_order MS:1002109 ! lower score better

[Term]
id: MS:1003130
name: ProSight:proteoform Q-value
def: "ProSight proteoform-level Q-value." [PSI:MS]
xref: value-type:xsd\:double "The allowed value-type for this CV term."
is_a: MS:1003127 ! proteoform-level Q-value
relationship: has_order MS:1002109 ! lower score better
```

**Comment by timosachsenberg** [comment]
Could it be that some id has been changed? e.g., the correct "proteoform-level Q-value" is now MS:1003129
```
[Term]
id: MS:1003129
name: proteoform-level Q-value
def: "Estimation of the Q-value for proteoforms." [PSI:MS]
xref: value-type:xsd\:double "The allowed value-type for this CV term."
is_a: MS:1002905 ! proteoform-level identification statistic
relationship: has_domain MS:1002305 ! value between 0 and 1 inclusive
relationship: has_order MS:1002109 ! lower score better
```

**Comment by chambm** [comment]
Fixed by #60
